advisories/a71655.md

```diff
@@ -12,13 +12,13 @@ Zigzag joins, which may be performed when two or more columns are held constant
 
 Incorrect results occur only when the indexes in question have multiple columns, the leading columns are held constant by the filter, and the trailing columns are the same amongst the indexes and are nullable. Incorrect results occur only when these nullable columns have null values.
 
-CockroachDB versions 19.2, 20.1, 20.2, and 21.1 are affected by this bug. Versions 19.2 and 20.1 are no longer eligible for [maintenance support](../releases/release-support-policy.html), so there will be no patches for these versions.
+CockroachDB versions 19.2, 20.1, 20.2, and 21.1 are affected by this bug. Version 20.2 is only affected until maintenance version 20.2.18. Versions 19.2 and 20.1 are no longer eligible for [maintenance support](../releases/release-support-policy.html), so there will be no patches for these versions.
 
 ## Statement
 
 This bug was resolved in CockroachDB by PR [71758](https://github.com/cockroachdb/cockroach/pull/71758), which corrects the bug by no longer planning zigzag joins when they produce incorrect results.
 
-The fix will be included in upcoming patches to 20.2 and 21.1. See [Mitigation](#mitigation) for the steps to avoid the issue until patches are released.
+The fix has been applied to maintenance version 20.2.18. The fix will be applied to an upcoming patch to 21.1. See [Mitigation](#mitigation) for the steps to avoid the issue until a patch is released.
```

Comment:
```suggestion
The fix has been applied to maintenance version 20.2.18. The fix will be applied to an upcoming maintenance release for 21.1. See [Mitigation](#mitigation) for the steps to avoid the issue until a patch is released.
```

Reply:
Fixed.